src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -182,19 +180,11 @@ private function hasProductWithPriceInChannel(array $products, int $price, Chann
             foreach ($product['variants'] as $variantIri) {
                 $this->client->executeCustomRequest(Request::custom($variantIri, HttpRequest::METHOD_GET));
 
-                /** @var array $channelPricings */
-                $channelPricings = $this->responseChecker->getValue($this->client->getLastResponse(), "channelPricings");
+                /** @var int $variantPrice */
+                $variantPrice = $this->responseChecker->getValue($this->client->getLastResponse(), "price");
```

Comment:
```suggestion
                $receivedPrice = $this->responseChecker->getValue($this->client->getLastResponse(), "price");
```

Reply:
the value we get is price of a variant i dont see reasons to name it received price :/ maybe receivedVariantPrice?